server/settings/publishers.py

```diff
@@ -1088,7 +1092,9 @@ class PublishersModel(BaseSettingsModel):
         "sync_workfile_version": False
     },
     "CollectFbxAnimation": {
-        "enabled": False
+        "enabled": True,
+        "optional": True,
+        "active": False
```

Comment:
Is it weird that we're now enabling both this and the other by default? I feel like this should still be disabled by default?
```suggestion
        "enabled": False,
        "optional": True,
        "active": False
```